jolt-core/src/jolt/vm/timestamp_range_check.rs

```diff
@@ -38,11 +39,11 @@ where
     C: CommitmentScheme<Field = F>,
 {
     _group: PhantomData<C>,
-    pub read_timestamps: [Vec<u64>; MEMORY_OPS_PER_INSTRUCTION],
-    pub read_cts_read_timestamp: [DensePolynomial<F>; MEMORY_OPS_PER_INSTRUCTION],
-    pub read_cts_global_minus_read: [DensePolynomial<F>; MEMORY_OPS_PER_INSTRUCTION],
-    pub final_cts_read_timestamp: [DensePolynomial<F>; MEMORY_OPS_PER_INSTRUCTION],
-    pub final_cts_global_minus_read: [DensePolynomial<F>; MEMORY_OPS_PER_INSTRUCTION],
+    pub read_timestamps: [Vec<u64>; REG_OPS_PER_INSTRUCTION + RAM_WORD_OPS_PER_INSTRUCTION],
```

Comment:
There is a variable called MEMORY_OPS_PER_INSTRUCTION = 7, which we use while generating the trace.
But REG_OPS_PER_INSTRUCTION + RAM_WORD_OPS_PER_INSTRUCTION = 5.
We can create a different variable for REG_OPS_PER_INSTRUCTION + RAM_WORD_OPS_PER_INSTRUCTION but it might be confusing.